### ISSUE TITLE
New Profile Showcase

### ISSUE THREAD

**Issue by LokeLok** [ValveSoftware/Dota2-Gameplay#12241]
### Description

Here are some things that can be considered for the new Profile Showcase.

----------------------------------
- [ ] Add the ability to Ctrl+C and Ctrl+V, to make it more convenient when you wanna add the same thing many times.
----------------------------------
- [ ] Add the ability to showcase Hero levels(Bronze, Silver, Gold, Platinum, Master Tier, Grandmaster). E.g. Show the number of Heroes you have in the Gold tier. 

![image](https://github.com/LokeLok/Dota-2-Ideas/assets/106764092/7ce49516-61d9-4ec5-b551-380d0df7c47c)
----------------------------------
- [ ] Add the ability to showcase the things that were behind 'Most Successful Heroes(All-Time)' Rampages, Stacked Camps, etc.
----------------------------------
- [ ] Emoticon tab should utilize the whole space instead of using a scroll bar.

![image](https://github.com/LokeLok/Dota-2-Ideas/assets/106764092/572edf58-cda0-4c8f-8011-83c6261125ad)
----------------------------------
- [ ] Profile badge should be under the Trophy tab(like it was before), so we can showcase it.

**Comment by Aphotic** [comment]
yeah, profile badge with a profile level was one of the fastest ways to detect new accounts smurfs make, now it's tough to judge.     
need a profile badge showcased obligatory or at least show profile level somehow when you hover over rank or name or something.  
have you made a reddit post to get upvotes here?

**Comment by Aphotic** [comment]
also while we are talking about this 
can we finally fix the "ability effects" of items?
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/30921064/daa03f42-788f-4da4-9ddc-f5b2aff0091b)

 the way items are shown currently on the new profile showcase/aftermatch are items that we are using for their ability effects and not the items we have equipped, it's a mismatch & defeats the purpose of it in the new profile showcase/aftermatch MPV screen & such.

**Comment by 3nines** [comment]
No we please, we don't need this useless shit again. The new profile helps the community remove unnecessary comments like "10,000 matches and still on legend". People with a negative attitude, like you, often scrutinize others' profiles to find something to criticize. Focus on your own match instead of staring at someone's profile. You can usually tell if someone is smurfing based on their farming and gameplay style.

**Comment by Aphotic** [comment]
#12326 created a new post just for the last case, it is important.

**Comment by LokeLok** [comment]
> No we please, we don't need this useless shit again. The new profile helps the community remove unnecessary comments like "10,000 matches and still on legend". People with a negative attitude, like you, often scrutinize others' profiles to find something to criticize. Focus on your own match instead of staring at someone's profile. You can usually tell if someone is smurfing based on their farming and gameplay style.

It should be an option to show it if you want and not by default. I would love to have the ability to put it on my Mini Profile.

**Comment by Aphotic** [comment]
> It should be an option to show it if you want and not by default. I would love to have the ability to put it on my Mini Profile.
 the badge optional? sure.
 but the life time games no, i think it should be shown somewhere obligatory since they introduced it last year to help find out smurf freshly created accounts that have very high WR/MMR

**Comment by Aphotic** [comment]
> It should be an option to show it if you want and not by default. I would love to have the ability to put it on my Mini Profile.

 the badge optional? sure.
 but the life time games no, i think it should be shown somewhere obligatory since they introduced it last year to help find out smurf freshly created accounts that have very high WR/MMR

**Comment by Aphotic** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/30921064/9980e726-e4a1-4e4b-81dd-cd65bd4998e4)

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/30921064/2722feb1-3a3a-41d1-883e-e6890e0a562d)
